spec/requests/form_controller_spec.rb

```diff
@@ -100,6 +109,16 @@
             expect(response).to have_http_status(:not_found)
           end
         end
+
+        context "when no other sections are enabled" do
+          before do
+            allow(in_progress_case_log.form).to receive(:next_incomplete_section_redirect_path).and_return("error")
```

Comment:
Does this need mocking? Shouldn't just using a case log with only 1 field answered using the 22/23 fixture work?

Reply:
done! Nice one